locale/en_US/common.xml

```diff
@@ -142,6 +142,7 @@
 	<message key="common.false">False</message>
 	<message key="common.file">File</message>
 	<message key="common.file.namingPattern">{$username}, {$userGroup}, {$originalFilename}</message>
+	<message key="common.file.anonymousNamingPattern">{$userGroup}, {$originalFilename}</message>
```

Comment:
Could you duplicate this throughout the other locales? Better to have it present (and accurate for most locales) than to have `##common.file.anonymousNamingPattern##` showing up as a universal default file title :)